example/gardener-local/controlplane/extensions-config/values.yaml.tmpl

```diff
@@ -0,0 +1,12 @@
+global:
+  # Please use the values.yaml in this folder to main your domain secrets
+  internalDomain:
+    provider: aws-route53
+    domain: internal.domain.local
+    zone: ""
+    credentials: {}
```

Comment:
That's true. I was following the pattern of `provider-local` setup, which is not ideal here, because the DNS provider config is not static. I'm open to better solutions 😄